src/platform/packages/shared/kbn-search-api-panels/components/eis_update_callout.tsx

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import React from 'react';
+import {
+  EuiButton,
+  EuiButtonIcon,
+  EuiFlexGroup,
+  EuiImage,
+  EuiLink,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import * as i18n from '../translations';
+import { useShowEisPromotionalContent } from '../hooks/use_show_eis_promotional_content';
+import searchRocketIcon from '../assets/search-rocket.svg';
+
+export interface EisUpdateCalloutProps {
+  ctaLink: string;
+  promoId: string;
+  isCloudEnabled: boolean;
+  handleOnClick: () => void;
+  direction: 'row' | 'column';
+  hasUpdatePrivileges: boolean | undefined;
+}
+
+export const EisUpdateCallout = ({
+  ctaLink,
+  promoId,
+  isCloudEnabled,
+  handleOnClick,
+  direction,
+  hasUpdatePrivileges,
+}: EisUpdateCalloutProps) => {
+  const { isPromoVisible, onDismissTour } = useShowEisPromotionalContent({
+    promoId: `${promoId}UpdateCallout`,
+    isCloudEnabled,
+  });
+
+  const dataId = `${promoId}-eis-update-callout`;
+
+  if (!isPromoVisible || !hasUpdatePrivileges) {
+    return null;
+  }
+
+  return (
+    <EuiPanel
+      grow={false}
+      data-telemetry-id={dataId}
+      data-test-subj={dataId}
+      css={({ euiTheme }) => ({
+        color: euiTheme.colors.primaryText,
+        border: `1px ${euiTheme.colors.borderBaseSubdued} solid`,
```

Comment:
would `border: euiTheme.border.thin,` work here? I think [euiTheme.border.thin](https://github.com/elastic/eui/blob/main/packages/eui-theme-borealis/src/variables/_borders.ts#L28-L31) the essentially the same as what you have